msteams-platform/m365-apps/publish.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Publish Teams apps for Microsoft 365
+description: Make your Microsoft 365-enabled Teams apps discoverable to users in Teams, Outlook, and Office
+ms.date: 05/24/2022
+ms.topic: conceptual
+ms.custom: m365apps
+ms.localizationpriority: medium
+---
+# Publish Teams apps for Microsoft 365
+
+Microsoft 365-enabled Teams apps are supported for production use in Microsoft Teams and can be distributed to preview audiences running the *Targeted Release* versions of outlook.com and office.com, and the *Beta Channel* build of Outlook for Windows desktop. Distribution options and processes for Microsoft 365-enabled Teams apps are the same as with traditional Teams apps.
+
+Once published, your app will be discoverable as an installable app from the Outlook and Office app stores in addition to Teams. When running in Outlook and Office, your app uses the same permissions granted in Teams. Teams admins can [Manage access to Teams apps across Microsoft 365](/MicrosoftTeams/manage-third-party-teams-apps) for users in their organization.
```

Comment:
Perhaps "in addition to the Teams app store" or the client Apps view or some such